bzt/modules/jmeter.py

```diff
@@ -130,7 +134,7 @@ def __set_jvm_properties(self):
         heap_size = self.settings.get("memory-xmx", None)
         if heap_size is not None:
             self.log.debug("Setting JVM heap size to %s", heap_size)
-            self._env["JVM_ARGS"] = "-Xmx%s" % heap_size
+            self._env["JVM_ARGS"] = os.environ.get("JVM_ARGS", "") + " " + "-Xmx%s" % heap_size
```

Comment:
if JVM_ARGS are empty, you'll get extra space, which might cause issues on windows. Better strip it.